Include/internal/pycore_pystate.h

```diff
@@ -43,6 +43,10 @@ struct _pending_calls {
     int last;
 };
 
+struct _opcodecache_config {
+    int minruns;
+};
```

Comment:
the struct is useless, just put directly the int into _ceval_runtime_state.

Reply:
I made a struct for easy extending more stuff in the config in the future.

I will remove it